f5lbaasdriver/v2/bigip/driver_v2.py

```diff
@@ -221,67 +221,28 @@ def _setup_crud(self, context, loadbalancer, entity, **kwargs):
     def _schedule_agent_and_device(self, context, loadbalancer,
```

Comment:
need to fix the travis issues. also this is a function used by many places, need to check if modifying this common function impacts the other interfaces, especially create lb.   or is it possible/better to create a separate schedule function for refresh/rebuild alone so that it does not affect existing interfaces?

Reply:
this is my original patch, https://github.com/F5Networks/f5-openstack-lbaasv2-driver/pull/1228. It passed every checking.

I will check the Travis and code. The code is too complicated to rebase.

Reply:
the Travis problem is caused by missing the `return agent, device` after I rebase the code.